CONTRIBUTING.md

```diff
@@ -10,6 +10,18 @@ of conduct can lead to a ban from the project and a report to GitHub.
 
 You can read the code of conduct [here](https://github.com/hikari-py/hikari/blob/master/CODE_OF_CONDUCT.md).
 
+# Versioning scheme
+
+This project follows the versioning scheme stated [here](https://semver.org/), but with a
+slight modification to the pre-release versioning: we denote all pre-releases with `.dev{num}`.
+
+The development version number is increased automatically after each release in master.
+
+# Deprecation process
+
+The removal or rename of anything facing the public facing API must undergo the a removal process, which should oblide to
```

Comment:
need to add this file to spell checking, "oblige" not "oblide" and even then that phrasing is weird since oblige would generally donate a legal or moral obligation.